packages/core/src/common/encoding-hex.ts

```diff
@@ -10,6 +10,7 @@ for (let n = 0; n <= 0xff; ++n) {
 export function hexToBytes(hex: string): Uint8Array {
   if (typeof hex !== 'string')
     throw new TypeError('hexToBytes: expected string, got ' + typeof hex);
+  if (hex.startsWith('0x')) hex = cleanHex(hex);
   if (hex.length % 2)
     throw new Error(`hexToBytes: received invalid unpadded hex, got: ${hex.length}`);
   const array = new Uint8Array(hex.length / 2);
```

Comment:
@whoabuddy Perhaps something along these lines would be a little closer to the micro-stacks coding style:

```suggestion
export function hexToBytes(hex: string): Uint8Array {
  if (typeof hex !== 'string')
    throw new TypeError('hexToBytes: expected string, got ' + typeof hex);
  const cleanedHex = cleanHex(hex); // already tolerant of both prefixed and non-prefixed args
  if (cleanedHex.length % 2)
    throw new Error(`hexToBytes: received invalid unpadded hex, got: ${cleanedHex.length}`);
  const array = new Uint8Array(cleanedHex.length / 2);
```

And of course use `cleanedHex` in remainder of fn as well.  Just my two cents.